### PR TITLE
da: Sync translation

### PR DESCRIPTION
**Pull request by mgeisler**
This resulted in some rewrapped strings due to how `msgmerge` wraps the strings compared to how `mdbook-xgettext` extracts them.

I also translated a few strings.

### CODE REVIEW

File: po/da.po [hunk 1]
```diff
@@ -1902,13 +1922,15 @@ msgstr ""
...
 msgid "# Welcome to Day 1"
-msgstr ""
+msgstr "# Velkommen til Dag 1"
 
 #: src/welcome-day-1.md:3
 msgid ""
 "This is the first day of Comprehensive Rust. We will cover a lot of ground\n"
 "today:"
 msgstr ""
+"Dette er den første dag af Comprehensive Rust. Vi vil gå over en samme\n"
```
Comment (thedataking) [inline po/da.po:1932]: Missing word here? Did you mean "Vi vil gå over en masse i dag"?

File: po/da.po [hunk 2]
```diff
@@ -1779,13 +1787,20 @@ msgid ""
...
+"}\n"
+"```"
 
 #: src/cargo/code-samples.md:19
 msgid ""
 "You can use <kbd>Ctrl + Enter</kbd> to execute the code when focus is in "
 "the\n"
 "text box."
 msgstr ""
+"Du kan bruge <kbd>Ctrl + Enter</kbd> for at udføre koden når fokus er\n"
```
Comment (thedataking) [inline po/da.po:1802]: nit: alternativt "afvikle"

File: po/da.po [hunk 3]
```diff
@@ -2002,7 +2025,7 @@ msgstr ""
 
 #: src/welcome-day-1/what-is-rust.md:21
 msgid "Rust fits in the same area as C++:"
-msgstr ""
+msgstr "Rust passer ind i det samme område som C++:"
```
Comment (thedataking) [inline po/da.po:2028]: What do you think about something like "Rust har det samme anvendelsesområde som C++? It wasn't immediately clear what "område" was referring to without looking at the original text.

File: po/da.po [hunk 4]
```diff
@@ -16241,496 +16337,228 @@ msgstr ""
...
-msgid "`src/controller.rs`:"
-msgstr ""
-
-#: src/exercises/concurrency/elevator.md:332
-msgid "<!-- File src/controller.rs -->"
-msgstr "<!-- File src/controller.rs -->"
+#: src/exercises/concurrency/chat-app.md:127
+#, fuzzy
+msgid "## Running the binaries"
+msgstr "Afvikling af kurset"
```
Comment (thedataking) [inline po/da.po:16522]: "kurset" vs "binaries" 🤔

**Review by thedataking — APPROVED** [review]
LGTM overall, left a few comments for you to consider.

File: po/da.po (hunk 1 above)
Comment (mgeisler) [reply po/da.po:1932]: Thanks you, definitely missed a word!

File: po/da.po (hunk 3 above)
Comment (mgeisler) [reply po/da.po:2028]: That is much better, thanks!

File: po/da.po (hunk 2 above)
Comment (mgeisler) [reply po/da.po:1802]: Thanks!

```suggestion
"Du kan bruge <kbd>Ctrl + Enter</kbd> for at afvikle koden når fokus er\n"
```

File: po/da.po (hunk 1 above)
Comment (mgeisler) [reply po/da.po:1932]:
```suggestion
"Dette er den første dag af Comprehensive Rust. Vi kommer til at dække en masse terræn\n"
```

**Pull request merged**